docs/resources/logs_restriction_query.md

```diff
@@ -0,0 +1,30 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_logs_restriction_query Resource - terraform-provider-datadog"
```

Comment:
In general this title is hard to read naturally, but ignore if it's typical or expected for this product.
```suggestion
page_title: "Datadog Log Restriction Query Resource"
```